packages/astro/src/core/encryption.ts

```diff
@@ -0,0 +1,68 @@
+import { base64, decodeHex, encodeHex } from 'oslo/encoding';
+
+export async function createKeyBytes() {
+	const key = await crypto.subtle.generateKey(
+		{
+			name: 'AES-GCM',
+			length: 256,
+		},
+		true,
+		['encrypt', 'decrypt']
+	);
+	const exported = await crypto.subtle.exportKey('raw', key);
+	return new Uint8Array(exported);
+}
+
+export async function createKey() {
```

Comment:
i don't think i need separate functions here, can just combine with createKeyBytes and not export in the former.

Reply:
Yep, that could make the internals simpler 👍